### PR TITLE
Adding konvoyconfig

### PR DESCRIPTION
**Pull request by jr0d**

### CODE REVIEW

File: templates/konvoyconfig.yaml
Comment (shaneutt) [inline templates/konvoyconfig.yaml]: :thinking: would we want to add a small comment here explaining to the reader the full context of how and why this configuration option works like this?

**Review by shaneutt — APPROVED** [review]
One non-blocking comment (but still might be worth updating for).

Marking this as blocked on https://github.com/mesosphere/charts/pull/141 as that needs to merge before this can.

**Comment by hectorj2f** [comment]
Yeah, let's wait until the PR is merged

File: templates/konvoyconfig.yaml
Comment (hectorj2f) [inline templates/konvoyconfig.yaml]: This version is 0.0.1

**Review by hectorj2f — CHANGES_REQUESTED** [review]

**Comment by jr0d** [comment]
@hectorj2f fixed

**Review by hectorj2f — APPROVED** [review]

**Pull request merged**
